modules/querier/querier.go

```diff
@@ -246,3 +252,168 @@ func (q *Querier) forGivenIngesters(ctx context.Context, replicationSet ring.Rep
 
 	return responses, err
 }
+
+func (q *Querier) Search(ctx context.Context, req *tempopb.SearchRequest) (*tempopb.SearchResponse, error) {
+	//userID, err := user.ExtractOrgID(ctx)
+	//if err != nil {
+	//	return nil, errors.Wrap(err, "error extracting org id in Querier.FindTraceByID")
+	//}
```

Comment:
I think this was copied from `Querier.FindTraceByID`. Since search doesn't need this, we probably commented it out to avoid compilation errors.
But now I'm wondering why we don't need `userID` in search 🤔

Reply:
Yeah we will need userID in search as well.

Reply:
Nah, we don't need this since we only query ingesters for now. The ingester can extract the userID out of `ctx` by itself.